### PR TITLE
 fix createLogoutTable method

### LINKED ISSUE #1323: Bug in createLogoutTable method
An error occurred while updating the simplesamlphp library

logs

>  4 [0900c986a2] Database error: array (#012  0 => '42601',#012  1 => 7,#012  2 => 'ERROR:  syntax error at or near "TIMESTAMP"#012LINE 1: ...impleSAMLphp_saml_LogoutStore ALTER COLUMN _expire TIMESTAMP#012

### PR DESCRIPTION
**Pull request by Marat555**
Closes https://github.com/simplesamlphp/simplesamlphp/issues/1323

### CODE REVIEW

File: modules/saml/lib/SP/LogoutStore.php [hunk 1]
```diff
@@ -40,7 +40,7 @@ private static function createLogoutTable(Store\SQL $store): void
                 case 'pgsql':
                     // This does not affect the NOT NULL constraint
                     $update = [
-                        'ALTER TABLE ' . $store->prefix . '_saml_LogoutStore ALTER COLUMN _expire TIMESTAMP'
+                        'ALTER TABLE ' . $store->prefix . '_saml_LogoutStore ALTER COLUMN type _expire TIMESTAMP'
```
Comment (tvdijen) [inline modules/saml/lib/SP/LogoutStore.php:43]: Shouldn't it be the other way around? `ALTER COLUMN _expire TYPE TIMESTAMP`?

Comment (Marat555) [reply modules/saml/lib/SP/LogoutStore.php]: Yes, it looks better.

**Comment by Marat555** [comment]
What's wrong with tests?

**Comment by tvdijen** [comment]
Nothing to worry about.. Sometimes I have to restart the Travis-tests... Scrutinizer is just a piece of junk that always fails for no reaosn

However, I think you have missed the point in my earlier comment.. I think the order of the words is wrong, unless I'm misinterpreting the docs..  The TYPE keyword should go _after_ the column name

**Comment by Marat555** [comment]
Fixed)

**Pull request merged**